pkgs/development/python-modules/pyxlsb/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyxlsb";
+  version = "1.0.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-gGLR6oYm0/GYDosc/pGkSDdHRJJC7LYQE7wt+FQ19oU=";
```

Comment:
Since the RHS is already an SRI hash (in the format `hashType-hash`), would you mind changing the LHS from `sha256` to `hash`? For transparency, this is not a must-do, as attempts to standardize on SRI hashes everywhere have failed in the past, but I feel like there is a popular preference to use `hash` on the LHS when the RHS _is_ an SRI hash.